plugin/pkg/scheduler/scheduler.go

```diff
@@ -92,6 +97,11 @@ func (s *Scheduler) scheduleOne() {
 		glog.V(1).Infof("Failed to schedule: %+v", pod)
 		s.config.Error(pod, err)
 		s.config.Recorder.Eventf(pod, api.EventTypeWarning, "FailedScheduling", "%v", err)
+		s.config.PodConditionUpdater.Update(pod, &api.PodCondition{
+			Type:   api.PodScheduled,
+			Status: api.ConditionFalse,
+			Reason: "Unschedulable",
```

Comment:
Can you add a field like `Message` to record the error message, in case users want to know the detail about why the pod is unschedulable:
```go
		s.config.PodConditionUpdater.Update(pod, &api.PodCondition{
			Type:   api.PodScheduled,
			Status: api.ConditionFalse,
			Reason: "Unschedulable",
			Message: err.Error(),
		})
```